src/PowerSimulations.jl

```diff
@@ -244,6 +244,7 @@ export PowerFlowVoltageAngle
 export PowerFlowVoltageMagnitude
 export PowerFlowLineReactivePowerFromTo, PowerFlowLineReactivePowerToFrom
 export PowerFlowLineActivePowerFromTo, PowerFlowLineActivePowerToFrom
+export PowerFlowLossFactors
```

Comment:
I've recently become aware that when we add new auxiliary variables we also have to add them to a documentation page. See https://github.com/NREL-Sienna/PowerSimulations.jl/pull/1248